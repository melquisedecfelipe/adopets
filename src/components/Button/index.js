import React from 'react';

import { Button } from 'antd';

import './styles.scss';

export default function ButtonComponent({ classe, label, onClick }) {
  return (
    <Button className={classe} onClick={onClick} type="submit">
      {label}
    </Button>
  );
}
