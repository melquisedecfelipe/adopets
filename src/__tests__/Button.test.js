import React from 'react';

import { shallow } from 'enzyme';

import Button from '../components/Button';

describe('Button testing with Enzyme', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Button />);

    expect(wrapper).toMatchSnapshot();
  });
});
