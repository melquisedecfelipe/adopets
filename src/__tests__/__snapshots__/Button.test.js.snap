// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button testing with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;
